compiler/noirc_driver/src/lib.rs

```diff
@@ -70,6 +70,9 @@ pub struct CompileOptions {
     #[arg(long, hide = true)]
     pub show_plonky2: bool,
 
+    #[arg(long, hide = true)]
+    pub write_plonky2_to_file: Option<String>,
```

Comment:
Rename to `plonky2_print_file`? The current name sounds like the name of a bool; so anything that sounds like the name of an (optional) file would be preferable.

Reply:
Done: https://github.com/blocksense-network/noir/pull/34/commits/29e1e050d57bd7cea169402f743d3a4ebba0d69d